### DISCUSSION TITLE
I cannot use GitHub or Github Desktop with my current IP address

### DISCUSSION THREAD

**Discussion by manumg35** [community/community#46329] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

This same afternoon I accessed github normally, now I can't even enter the website.
Also, when I try to clone a repository using GitHub Desktop this error appears "The host is down. Check your Internet connection and try again."
I dont know why but if I use a VPN I can normally use the website, I guess something is wrong with my IP address but I dont know what to do... :(
Thanks in advance.

**Comment by jge162** [comment, marked as answer]
sounds like your IP address may have been temporarily blocked by GitHub. If you believe this is a mistake, you can contact -> [GitHub Support]( https://github.com/contact) and select “Report a website issue".
